helm-chart/todoapp/charts/mysql/values.yaml

```diff
@@ -0,0 +1,42 @@
+---
+common:
+  namespace: mysql
+
+mysql:
+  image:
+    name: mysql
+    repository: mysql
+    version: 8.0
+
+  resources:
+    requests:
+      cpu: 500m
+      memory: 1Gi
+
+  statefulSet:
+    replicas: 2
+    resources:
+      storage: 2Gi
+
+  affinity:
+    pod:
+      key: "app"
+      operator: In
+      values: "mysql"
+      topologyKey: "kubernetes.io/hostname"
+    nod:
```

Comment:
There seems to be a typo in the key `nod`. It should likely be `node` to correctly represent node affinity settings.